pygmt/src/info.py

```diff
@@ -88,7 +86,9 @@ def info(
         - :class:`numpy.ndarray` if either of the above parameters are used.
         - str if none of the above parameters are used.
     """
-    aliasdict = AliasSystem().add_common(
+    aliasdict = AliasSystem(
+        I=Alias(spacing, name="spacing", sep="/"),
```

Comment:
Need to note that `spacing` can be a sequence of two or more values, so we don't check its size here.